lib/database/sql/000017_peer_count_shard_origin.up.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE IF NOT EXISTS peerCountByShard (
+    id SERIAL PRIMARY KEY,
+    recordId INTEGER NOT NULL,
+    count INTEGER NOT NULL,
+    shard INTEGER NOT NULL,
+    timestamp INTEGER NOT NULL,
+    CONSTRAINT peerCountByShard_unique UNIQUE (recordId, count, shard, timestamp)
+);
+
+ALTER TABLE peerCountByShard ADD CONSTRAINT fk_peerCountByShard_telemetryRecord
+            FOREIGN KEY (recordId) REFERENCES telemetryRecord(id);
+
+CREATE TABLE IF NOT EXISTS peerCountByOrigin (
+    id SERIAL PRIMARY KEY,
+    recordId INTEGER NOT NULL,
+    count INTEGER NOT NULL,
+    origin INTEGER NOT NULL,
+    timestamp INTEGER NOT NULL,
+    CONSTRAINT peerCountByOrigin_unique UNIQUE (recordId, count, origin, timestamp)
+);
+
+ALTER TABLE peerCountByOrigin ADD CONSTRAINT fk_peerCountByOrigin_telemetryRecord
+            FOREIGN KEY (recordId) REFERENCES telemetryRecord(id);
+
+
+SELECT * FROM peerCountByShard ORDER BY timestamp DESC LIMIT 10;
+SELECT * FROM peerCountByOrigin ORDER BY timestamp DESC LIMIT 1;
```

Comment:
What's this for? :thinking:

Reply:
leftover from using IDE to build query, removed